### PR TITLE
replace var with explicit types

### PR DESCRIPTION
**Pull request by kelseyduffy**

### CODE REVIEW

**Review by anton-dahlin — APPROVED** [review]

File: SdsTsDotNet/Program.cs [hunk 1]
```diff
@@ -87,7 +87,7 @@ public static async Task<bool> MainAsync(bool test = false)
                 };
                 pressure_stream = await metadataService.GetOrCreateStreamAsync(pressure_stream).ConfigureAwait(false);
```
Comment (derekendres) [inline SdsTsDotNet/Program.cs:89]: var missed on line 82

**Review by derekendres — CHANGES_REQUESTED** [review]
var missed on line 82.  also there are 9 messages about simplifying new statements.  This was also listed by Brandon to do.

File: SdsTsDotNet/Program.cs (hunk 1 above)
Comment (kelseyduffy) [reply SdsTsDotNet/Program.cs:89]: good catch, I missed the new () statement being simplified in his email. I didn't know that was even a thing. They should all be changed now

**Review by derekendres — APPROVED** [review]

File: SdsTsDotNet/Program.cs (hunk 1 above)
Comment (derekendres) [reply SdsTsDotNet/Program.cs:89]: yeah, i have mixed feeling on the new simplification.  I think it is cleaner and neat overall, but i haven't seen it that much so it feels odd to me still

**Review by cbardini — APPROVED** [review]

**Pull request merged**